istqb-ctal-ta-complete-guide.md

```
@@ -10,23 +10,23 @@
-1. [CTAL-TA v4.0 概要](#chapter-0-ctal-ta-v40-概要)
-2. [Chapter 1: テストプロセスにおけるテストアナリストのタスク](#chapter-1-テストプロセスにおけるテストアナリストのタスク225分)
-3. [Chapter 2: リスクベーステストにおけるテストアナリストのタスク](#chapter-2-リスクベーステストにおけるテストアナリストのタスク90分)
-4. [Chapter 3: テスト分析とテスト設計](#chapter-3-テスト分析とテスト設計615分)
-5. [Chapter 4: 品質特性のテスト](#chapter-4-品質特性のテスト60分)
-6. [Chapter 5: ソフトウェア欠陥防止](#chapter-5-ソフトウェア欠陥防止225分)
-7. [試験対策・サンプル問題](#試験対策サンプル問題)
-8. [v3.1 → v4.0 主な変更点](#v31-v40-主な変更点)
-9. [参照URL一覧](#参照url一覧)
+1. [CTAL-TA v4.0 概要](#-chapter-0-ctal-ta-v40-概要)
+2. [Chapter 1: テストプロセスにおけるテストアナリストのタスク](#-chapter-1-テストプロセスにおけるテストアナリストのタスク225分)
+3. [Chapter 2: リスクベーステストにおけるテストアナリストのタスク](#-chapter-2-リスクベーステストにおけるテストアナリストのタスク90分)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)